### PR TITLE
[CLI] Create "update:check-new-version" command

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Create "update:check-new-version" command
| Type?             |new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | -
| How to test?      | use new `update:check-new-version` command

### CODE REVIEW

File: classes/Commands/CheckNewVersionCommand.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+     * @throws ZipActionException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): ?int
+    {
+        try {
+            $this->setupContainer($input, $output);
+            $this->upgradeContainer->initPrestaShopAutoloader();
+            $this->upgradeContainer->initPrestaShopCore();
+
+            $this->upgradeContainer->getState()->initDefault(
```
Comment (ga-devfront) [inline classes/Commands/CheckNewVersionCommand.php]: we going to create a state file only to get current bersion but you already pass it to this method init with `$this->upgradeContainer->getProperty(UpgradeContainer::PS_VERSION)` I think we can remove this part and pass directly $this->upgradeContainer->getProperty(UpgradeContainer::PS_VERSION)

**Review by ga-devfront — DISMISSED** [review]
I think it's ok without the little comment I left

File: classes/Commands/CheckNewVersionCommand.php (hunk 1 above)
Comment (M0rgan01) [reply classes/Commands/CheckNewVersionCommand.php]: this part was not necessary for the process, I removed it

**Review by ga-devfront — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
Just found out an issue:

image

The online channel should be first in the list, whatever its destination version compared to the local updates.

**Review by ga-devfront — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @M0rgan01 

Thank you for your PR, I tested it and it seems to works as you can see : 

image

Tested from : 
1.7.8.8 to 8.2 
8.1.4 to 8.2 
8.2 to 9.0.0

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Pull request merged**